### PR TITLE
fixed and added the option to check for default sentences

### PR DESCRIPTION
**Pull request by iyeshaya**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/33031

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/readme.py [hunk 1]
```diff
@@ -234,20 +238,44 @@ def verify_no_empty_sections(self) -> bool:
...
+            sections_list - list of strings, each string is a section to find in the text
+            ignore_packs - List of packs and integration names to be ignored
+        Returns:
+            An error message with the relevant sections.
+        """
+        errors = ""
+
+        current_pack_name = self.pack_path.name
+        if ignore_packs and current_pack_name in ignore_packs:
+            print("\n\n ignoring this pack \n\n")
```
Comment (roysagi) [inline demisto_sdk/commands/common/hook_validations/readme.py:254]: please use `click.secho` and print this in yellow as this is a warning.

File: demisto_sdk/commands/common/hook_validations/readme.py [hunk 2]
```diff
@@ -234,20 +238,44 @@ def verify_no_empty_sections(self) -> bool:
...
-            required_section = re.findall(rf'{section}', self.readme_content, re.IGNORECASE)
-            if required_section:
-                errors += f'Replace "{section}" with a suitable info.\n'
-                is_valid = False
+        errors += self._find_section_in_text(USER_FILL_SECTIONS)
+        errors += self._find_section_in_text(DEFAULT_SENTENCES, PACKS_TO_IGNORE)
+        is_valid = not bool(errors)
+        print("===================================================\n")
+        print(errors)
+        print("\n===================================================")
```
Comment (roysagi) [inline demisto_sdk/commands/common/hook_validations/readme.py:278]: I think these are debug prints right?

**Review by roysagi — CHANGES_REQUESTED** [review]
If you could please share a print screen of the resulting error here.

File: demisto_sdk/commands/common/hook_validations/readme.py (hunk 2 above)
Comment (iyeshaya) [reply demisto_sdk/commands/common/hook_validations/readme.py]: right, removed

File: demisto_sdk/commands/common/hook_validations/readme.py (hunk 1 above)
Comment (iyeshaya) [reply demisto_sdk/commands/common/hook_validations/readme.py]: Done

**Review by roysagi — APPROVED** [review]
Please add a changelog entry

**Pull request merged**